src/host/main.cpp

```diff
@@ -440,6 +440,11 @@ int main(int argc, char** argv)
       }
       LOG_INFO_FMT("Reading previous service identity from {}", idf);
       startup_config.recover.previous_service_identity = files::slurp(idf);
+      while (startup_config.recover.previous_service_identity->back() == 0x00)
```

Comment:
Could these be removed before this PR is merged?

Reply:
Yes, I want to remove those too. @eddyashton last chance if you want to keep them, but I don't think you do.

Reply:
Sorry I missed this - yeah I think those should be removed, thanks.